Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -249,16 +185,14 @@ void TRDGlobalTracking::endOfStream(EndOfStreamContext& ec)
        mTimer.CpuTime(), mTimer.RealTime(), mTimer.Counter() - 1);
 }
 
-DataProcessorSpec getTRDGlobalTrackingSpec(bool useMC, bool useTrkltTransf, GTrackID::mask_t src)
+DataProcessorSpec getTRDGlobalTrackingSpec(bool useMC, GTrackID::mask_t src)
 {
   std::vector<OutputSpec> outputs;
 
   dataRequestTRD.requestTracks(src, false);
```

Comment:
One more point. I don't really like that dataRequestTRD is a global variable.
This can be done more elegantly using a shared ptr, as I have done e.g. in https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/Workflow/src/O2GPUDPLDisplay.cxx#L111
(Ping @shahor02 FYI)

Reply:
Done